net/gnunet/test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+gnunet-arm --version
```

Comment:
```suggestion
gnunet-arm --version | grep "$2"
```

This way it checks if the current version is actually printed, not just if something (e.g. Segfault) is printed.